clerk-tools.lic

```diff
@@ -0,0 +1,91 @@
+
+custom_require.call(%w[drinfomon equipmanager common common-crafting common-items common-travel])
+
+class Clerk
+  include DRC
+  include DRCC
+  include DRCI
+  include DRCT
+
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'toolset', options: %w[forging outfitting engineering], description: 'What set of tools to use' },
+        { name: 'action', options: %w[get store], description: 'Whether to get tools, or store them with the clerk' },
+      ]
+    ]
+    args = parse_args(arg_definitions)
+    action = args.action
+
+    settings = get_settings
+    hometown = settings.hometown
+    @bag = settings.crafting_container
+    @belt = settings.engineering_belt
+    
+    crafting_data = get_data('crafting')
+
+    if args.toolset == 'engineering'
+      tools = settings.shaping_tools
+      area = 'shaping'
+    end
+
+    if args.toolset == 'outfitting'
+      tools = settings.outfitting_tools
+      area = 'tailoring'
+    end
+
+    if args.toolset == 'forging'
+      tools = settings.forging_tools
+      area = 'blacksmithing'
+    end
+
+    if args.toolset == 'alchemy'
+      tools = settings.alchemy_tools
+	  area = 'remedies'
+    end
+
+    roomnumber = crafting_data[area][hometown]['repair-room']
+    repairnpc = crafting_data[area][hometown]['repair-npc']
+
+    if action=="store"
+      storeTools(tools,roomnumber)
+    elsif action="get"
+      getTools(tools,roomnumber,repairnpc)
+    else
+      echo 'Unknown action'
+    end
+  end
+  
+  def getTools(tools,roomnumber,repairnpc)
+    DRCT.walk_to(roomnumber)
+    tools.each do | tool |
+      case bput("ask #{repairnpc} for #{tool}",'Ah, yes, we have one of your tools like that','Ah, yes, we have several of your tools like that',"It doesn't look like we have anything like that")
+      when 'Ah, yes, we have one of your tools like that'
+        stow_crafting_item(tool, @bag, @belt)
+      when 'Ah, yes, we have several of your tools like that'
+        ##Sack
+      end
+    end
+  end
+  
+  def storeTools(tools,roomnumber)
+    echo roomnumber
```

Comment:
💡 This might be more useful to the user if there was some context, e.g. `echo "roomnumber: #{roomnumber}"`